packages/amplify-category-auth/src/index.js

```diff
@@ -16,8 +16,14 @@ const {
   removeDeprecatedProps,
 } = require('./provider-utils/awscloudformation');
 
-const { transformUserPoolGroupSchema } = require('./utils/transform-user-pool-group');
+const { transformUserPoolGroupSchema } = require('./provider-utils/awscloudformation/utils/transform-user-pool-group');
 const { uploadFiles } = require('./provider-utils/awscloudformation/utils/trigger-file-uploader');
+const { validateAddAuthRequest } = require('amplify-util-headless-input');
+const {
+  getAddAuthRequestAdaptor: addAuthRequestAdaptorFactory,
```

Comment:
if we need aliasing the import shouldn't we rename the export itself? semantically the `get` and `add` is somehow different, get returns something, add adds something to a passed in array. At least that is what I'd predict from the naming.

Reply:
This was left over from an automatic rename. Fixed